docs/usecase/usage-example.md

```diff
@@ -0,0 +1,62 @@
+# Use cases
+
+This document will introduce you the example of what Vald can do.
+Vald is the approximate nearest neighbor dense vector search engine. Vald supports the general usecase of similarity search engine, like faiss.
+
+Here is the example of the use cases of Vald.
+
+1. Image and video processing
+
+    You can use Vald as the image/video processing engine to search the similar image/video or analysis the image/video for your use case.
+
+    Vald is capable to process huge number of images at the same time so it should fit with your use case.
+
+    Here are some example of what you can do with images and video in Vald.
+
+    1. Search by image
+    1. Face recognition
+    1. Product recommandation base on images
+    1. Image/Video analysis
+    1. Image/Video deduplication
+
+    In addition Vald support user-defined filter that user can customize the filter to filter the specific result.
+
+    For example when user chose a man's t-shirt and the similar product is going to be searched in Vald.
+
+    Without the filtering functionality the women's t-shirt may be searched in Vald and displayed because women's t-shirt is similar to the men's t-shirt and it is very hard to differentiate the image of men's and women's t-shirt.
```

Comment:
```suggestion
    Without the filtering functionality, the women's t-shirt may be searched in Vald and displayed because women's t-shirt is similar to the men's t-shirt and it is very hard to differentiate the image of men's and women's t-shirt.
```